pkg/core/resources/apis/mesh/validators.go

```diff
@@ -18,6 +18,7 @@ type TagValueValidatorFunc func(path validators.PathBuilder, key, value string)
 
 type ValidateSelectorOpts struct {
 	SkipRequireAtLeastOneTag bool
+	SkipRequireService       bool
```

Comment:
Maybe, let's invert `SkipRequireService` into `RequireServiceTag` ?

And refactor `SkipRequireAtLeastOneTag` into `RequireAtLeastOneTag` in this or separate PR ?